gunicorn/arbiter.py

```diff
@@ -302,6 +299,23 @@ def handle_winch(self):
         else:
             self.log.debug("SIGWINCH ignored. Not daemonized")
 
+    def if_promoted(self):
```

Comment:
I would give this a new name. Maybe `manage_arbiters` for symmetry with `manage_workers`. Right now it looks to me like a function that returns a boolean value but doesn't have side effects.